AirCastingTests/API/SyncContextServiceTests.swift

```diff
@@ -0,0 +1,171 @@
+// Created by Lunar on 19/06/2021.
+//
+
+import XCTest
+import Combine
+@testable import AirCasting
+
+class SyncContextServiceTests: XCTestCase {
+    let validContextResponseData: Data =
+        """
+        {
+            "download": ["UUID-TO-DOWNLOAD1", "UUID-TO-DOWNLOAD2"],
+            "upload": ["UUID-TO-UPLOAD1", "UUID-TO-UPLOAD2"],
+            "deleted": ["UUID-TO-DELETE1", "UUID-TO-DELETE2"],
+        }
+        """.data(using: .utf8)!
+    let client = APIClientMock()
+    let auth = RequestAuthorizationServiceMock()
+    let responseValidator = HTTPResponseValidatorMock()
+    lazy var service = SessionSynchronizationService(client: client, authorization: auth, responseValidator: responseValidator)
+    private var cancellables: [AnyCancellable] = []
```

Comment:
Make private things private. 
Also XCtest allocate a new instance for each test method and delicate all when all test pass. So please use teardown to remove unnecessary objects. I use a lazy var with declaring the type to be force unwrapped optional so teardown only sets the values to nil and that's it. Also not clearing this values can lead to false positives.

Reply:
I don't think in this case we need any cleanup here. I'll cancel all cancellables in a `tearDown` just to be sure, but honestly think it's redundant